### ISSUE TITLE
VAD to detect simultaneous music and voice

### ISSUE THREAD

**Issue by realies** [ina-foss/inaSpeechSegmenter#48]
This is more of a feature request - is it possible to detect simultaneous music and voice?

**Comment by DavidDoukhan** [comment]
Right now, this would require to design new voice activity detection systems within inaspeechsegmenter.
Are you aware of corpora allowing to design and evaluate such systems ?

**Comment by realies** [comment]
Not really. I presumed the preexisting functionality and datasets can be changed to distinguish between music and music with narration over it, based on some confidence ratio. Your comment makes it sound like to achieve this, the project needs a completely different VAD system?

**Comment by realies** [comment]
@DavidDoukhan, could existing corpora be used to mix music and voice with various ratios and extend the training dataset in a new VAD mode?

**Comment by realies** [comment]
@DavidDoukhan, is this really completed?

**Comment by DavidDoukhan** [comment]
This won't be done.
